### ISSUE TITLE
about the d z_tilde / d logits

### ISSUE THREAD

**Issue by thudzj** [duvenaud/relax#6]
I found that in the Appendix B of paper, v' can still differentiate with respect to \theta but the code (https://github.com/duvenaud/relax/blob/master/pytorch_toy.py#L105) omits it. Why? Because the effect of it is small?

**Comment by sdrobert** [comment]
@wgrathwohl I think @thudzj is right, and I've made a slight mistake in my implementation. If you think so, too, I'd be happy to make a PR with the change.

Sorry and thanks!

**Comment by agadetsky** [comment]
@sdrobert did you make a PR?

**Comment by sdrobert** [comment]
@agadetsky I've just submitted the PR. AFAICT it's working well now and converges a bit faster. Please note that the pytorch implementation was all my doing and thus the bug is entirely my fault. Sorry for the trouble.

Though I've included the concrete relaxation as preprocessing `z` and `z_tilde` in the RELAX estimator because every other source in the repository does this, it is not integral to RELAX. Consider removing `sig_z` and `sig_z_tilde` from the implementation, depending on whether you converge more quickly with it.

A quick plug: I've since written up RELAX and other estimators in [pydrobert-pytorch](https://pydrobert-pytorch.readthedocs.io/en/stable/). The interface is much more flexible, there's support for categorical variables, and I've written multiple tests to verify the integrity of the estimators. The package is available through PyPI and Conda.

**Comment by agadetsky** [comment]
@sdrobert Thank you very much! Do you have any examples for using pydrobert-pytorch?

**Comment by agadetsky** [comment]
@sdrobert If you have any example training RELAX using your library can you share it?

**Comment by sdrobert** [comment]
@agadetsky A cookbook is definitely on my to-do list. Currently, I'm adding more functionality according to the experimentation I'm doing. The best you'll get right now are in my unit tests:
[here](https://github.com/sdrobert/pydrobert-pytorch/blob/master/tests/test_estimators.py#L154) and [here](https://github.com/sdrobert/pydrobert-pytorch/blob/086c0f80ba72e7bd53cfdbdcd4eb4a13b7da64f6/tests/test_estimators.py#L86).

**Comment by agadetsky** [comment]
@sdrobert I get it, thank you, I think it will be enough.
